examples/sha/src/main.rs

```diff
@@ -18,18 +18,27 @@ use risc0_zkvm::{
     sha::Digest,
     Executor, ExecutorEnv,
 };
-use sha_methods::HASH_ELF;
+use sha_methods::{HASH_ELF, HASH_RUST_CRYPTO_ELF};
 
-fn provably_hash(input: &str) -> Digest {
+fn provably_hash(input: &str, use_rust_crypto: bool) -> Digest {
     let env = ExecutorEnv::builder()
         .add_input(&to_vec(input).unwrap())
         .build();
 
-    let mut exec = Executor::from_elf(env, HASH_ELF).unwrap();
+    let elf = if use_rust_crypto {
```

Comment:
Maybe add a comment here saying that this enables/disables circuit acceleration patching for the RustCrypto sha2 crate by switching between guest code that does/doesn't use it; and probably also directing people to the corresponding guest files.